CodeReview/src/nodes/CommentedNode.h

```diff
@@ -45,12 +48,13 @@ class CODEREVIEW_API CommentedNode : public Super<Model::CompositeNode>
 	COMPOSITENODE_DECLARE_STANDARD_METHODS(CommentedNode)
 
 	ATTRIBUTE(Model::Text, nodeId, setNodeId)
+	ATTRIBUTE(Model::Text, managerName, setManagerName)
```

Comment:
Dont' use just `ATTRIBUTE`, instead use `ATTRIBUTE_VALUE` as in for example is done in `Comments::CommentFreeNode`. That makes it so that you don't have to always use `set` and `get` but can directly work with the value.